File: foundation/foundation 3 month/lesson4/homework/ex_13.py
import random
k=int(input("K ni kiriting: "))
arr=list()
for x in range(k):
	arr.append(random.randint(10,30))
for x in arr:
	print(x,end=",")
j=0
while j<len(arr):
	if arr[j]%2==0:
		arr.pop(j)
	else:
		j+=1
print("")
for x in arr:
	print(x,end=",")

File: full stack/full stack 1 month/lesson5/homework/ex_13.js
let n = 5
let arr =[]

for (let i = 1; i<=n; i++)
{
    arr.push(i)
}
let cnt = 0
for (let i = 0; i <arr.length; i++)
{
    if (arr[i] %2 == 1)
    {
       console.log(arr[i])
        cnt+=1
    }
}

console.log(`Cnt: ${cnt}`)
